DirectEmployers/MyJobs
# Pd 2615 Load existing contact data; attempt 2

## What's this PR do?

* When selecting an existing contact in NUO, show all the data.
* Fix the notes append function, which was broken.

## Screenshots (if appropriate)

![screenshot](https://cloud.githubusercontent.com/assets/108186/23569340/c75b9154-002c-11e7-9eb1-d7650511c9b6.png)

## Where should the reviewer start?

Look at `forms.py`, then `views.py`.

## How should this be manually tested?

1. Go to the NUO app.
2. Search for a partner and any contact associated with that partner OR create a new contact and enter information for that contact inside of the contact details page. 
3. When you submit the form after creating a new contact OR pulling down an existing contact, clicking on the contacts name on the right hand facet side should bring up the contact details form to append notes to the form which should also have the existing entered data in there from before instead of being blank.

## Any background context you want to provide?

I've tried to chip away at the complexity of the `api_convert_outreach_record` view by making the forms do more and the view do less.
